service/history/replication/dlq_handler.go

```diff
@@ -219,7 +219,7 @@ func (r *dlqHandlerImpl) MergeMessages(
 		}
 	}
 
-	err = r.shard.GetExecutionManager().RangeDeleteReplicationTaskFromDLQ(
+	_, err = r.shard.GetExecutionManager().RangeDeleteReplicationTaskFromDLQ(
```

Comment:
we don't need to specify the batch size and check the result here?

Reply:
this is called in an api handler, i don't want to make it fail in the middle. it make break the semantics of the api